compiler/rustc_arena/src/lib.rs

```diff
@@ -11,10 +11,12 @@
     html_root_url = "https://doc.rust-lang.org/nightly/nightly-rustc/",
     test(no_crate_inject, attr(deny(warnings)))
 )]
+#![allow(incomplete_features)]
 #![feature(dropck_eyepatch)]
 #![feature(new_uninit)]
 #![feature(maybe_uninit_slice)]
-#![feature(min_specialization)]
+//#![feature(min_specialization)]
+#![feature(specialization)]
```

Comment:
Full specialization is unsound, so this will probably not be accepted.

Reply:
Thx. Worked around with `min_specialization`.